src/utils/contract.ts

```diff
@@ -102,8 +102,8 @@ export default class Contract extends BaseConnection {
     ] as const;
     if (this.provider) {
       return await this.provider
-        .sendAsync('eth_call', params)
-        .then(resp => resp.json());
+      .sendAsync('eth_call', params)
+      .then(resp => resp.json !== undefined ? resp.json() : resp);
```

Comment:
I am not sure if this is the best way to solve the issue but after this change, the test are being passed.

Reply:
You need to solve this when you make a wrapper inside `Resolution.fromWeb3Provider`.